packages/p2p-v2/src/pages/my-profile/screens/MyProfileStats/MyProfileStatsItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import './MyProfileStatsItem.scss';
+
+type MyProfileStatsItemProps = {
+    currency?: string;
+    label: string;
+    onClickLifetime?: (isLifetimeClicked: boolean) => void;
+    shouldShowDuration?: boolean;
+    shouldShowLifetime?: boolean;
+    value: string;
+};
+const MyProfileStatsItem = ({
+    currency,
+    label,
+    onClickLifetime,
+    shouldShowDuration = true,
+    shouldShowLifetime,
+    value,
+}: MyProfileStatsItemProps) => {
+    const [hasClickedLifetime, setHasClickedLifetime] = useState(false);
+
+    return (
+        <div className='p2p-v2-my-profile-stats__item'>
+            <span>
+                {label}{' '}
+                {shouldShowDuration && (
+                    <button
+                        className={clsx(
+                            'p2p-v2-my-profile-stats__item--inactive',
+                            !hasClickedLifetime && shouldShowLifetime && 'p2p-v2-my-profile-stats__item--active'
+                        )}
```

Comment:
why not this
`className={clsx(
                            'p2p-v2-my-profile-stats__item--inactive',
                             'p2p-v2-my-profile-stats__item--active': !hasClickedLifetime && shouldShowLifetime
                        )}`
instead?